api/src/main/java/com/gentics/mesh/etc/config/AuthenticationOptions.java

```diff
@@ -25,8 +25,12 @@
 public class AuthenticationOptions implements Option {
 
 	public static final String DEFAULT_ALGORITHM = "HS256";
+	public static final String DEFAULT_ISSUER = "Gentics Mesh";
```

Comment:
Since we in Mesh do not tell apart the client token and API token, this is also a breaking change regarding API token usage, which now awaits for the issuer+audience in payload during the API call. I would default both new fields to null.

Reply:
I tested this scenario, and it is not breaking.
The JWT Authenticator does not automatically verify the received `Issuer` or `Audience` fields.
It's also stated in the JWT reference that these fields are optional and should be checked on an Application Layer instead.

I also think the information of `Issuer` and `Audience` is not really relevant for Mesh itself, but for clients/applications using Mesh.
The only place where this would be relevant imo would be in OAuth2 handlers, where we validate the token from another service (i.E. Keycloak).